jwst/transforms/resources/schemas/stsci.edu/jwst_pipeline/gwa_to_slit-1.0.0.yaml

```diff
@@ -28,12 +28,12 @@ allOf:
               anyOf:
                 - type: array
                 - type: number
-          - $ref: ../asdf/core/ndarray-1.0.0
+          - $ref: "http://stsci.edu/schemas/asdf/core/ndarray-1.0.0"
       models:
         description: |
           A compound model transferring positions at the GWA to
           position in the slit frame.
         type: array
         items:
-          $ref: "../asdf/transform/transform-1.1.0"
+          $ref: "http://stsci.edu/schemas/asdf/transform/transform-1.2.0"
```

Comment:
This is one of the discussion questions in AL-572, but I wonder how valuable it is to `$ref` the transform-1.2.0 schema in situations like this.  What we really want to assert is that the object is some type of transform, but all the transform-1.2.0 schema does is cursory and optional validations of the "name" and "inverse" properties.  Any object without a "name"/"inverse" would qualify!

So the `$ref` doesn't buy us anything, except to show people reading the schema that a transform goes here.  That being the case we could just include that information in the description, and not `$ref` any schema.  This saves us from having to update the URI here when a new transform-x.y.z is released.

Reply:
OK, so you are suggesting removing `items`.